language/vm/vm_runtime/vm_runtime_types/src/value.rs

```diff
@@ -79,30 +81,30 @@ enum ValueImpl {
 /// A Move value.
 ///
 /// `Value` is just a wrapper type around `[ValueImpl]` which allows only Move types.
-#[derive(PartialEq, Eq, Debug, Clone)]
+#[derive(PartialEq, Eq, Debug, Clone, Serialize, Deserialize)]
```

Comment:
those De/Ser should not be needed and you can remove them as we talked about

Reply:
Turns out leaving the Serialize impl on MutValue simplifies things greatly so I'll remove all of them but that one